sintr_worker/bin/worker_isolate.dart

```diff
@@ -52,30 +50,20 @@ Future<String> _protectedHandle(String msg) async {
 
 // Sample extractor
 
-Future<Map<String, List<String>>> map(String k, String v) async {
+Future<Map<String, List<String>>> map(String logKey) async {
```

Comment:
Just a key.

This is a general framework that doesn't know that it's processing logs, and indeed it might well not be.